Dan/problem-2-solutionv2.js

```diff
@@ -0,0 +1,94 @@
+
+//let calls = 0
+function solution(elevationList){
```

Comment:
Having a little hard time understanding your solution. Maybe it could be my pre-conceived notion of the solution and lack of understanding on javascript. I would love to know your algorithm - will discuss it in the meeting.

Reply:
sounds great cause I'm having a tough time wrapping my head around the way you solved it. I definitely struggled with this one a bit